.travis-build-without-test.sh

```diff
@@ -31,8 +31,8 @@ if [ -d ../annotation-tools ] ; then
     echo "... done: (cd ../annotation-tools && git pull)"
 else
     set +e
-    echo "Running: git ls-remote https://github.com/${SLUGOWNER}/annotation-tools.git &>-"
-    git ls-remote https://github.com/${SLUGOWNER}/annotation-tools.git &>-
+    echo "Running: wget https://github.com/${SLUGOWNER}/annotation-tools.git &>-"
```

Comment:
Please change all calls of `wget` to `wget  -q --spider`. Put that in a variable to not copy many times.
This will only check for the file, not actually download it.

Note that the PR is currently failing a Travis test, b/c the downloaded files contain trailing whitespaces.
In general we don't want these extra files lying around, so let's change the command to not do that.